### ISSUE TITLE
 Role and Label missing , Design issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#833]
**Describe the bug**
Role and Label missing for Anchor tags.

**To Reproduce**

Go to Design  (http://localhost:9992/cm/app/?view=design&pathType=unknown)
Click on a site.
Under Action menu, the options like "Create template from page", "Add template" etc , have no Role and Label attributes, when inspecting them in browser.

**Expected behavior**
Role and Label should be present for Anchor tags.

**Additional context**
Navigate all the options available for similar issue and fix.

**Comment by ramdev1980** [comment]
Site and page table in top section of page is not getting fixed, after making changes in files.

**Comment by ramdev1980** [comment]
Could not reproduced. Seems it got fixed by some other WCAG issue. So closing it.
